lambdas/service/app.py

```diff
@@ -2233,7 +2287,50 @@ def get_data_object(file_uuid):
         return Response(response.text, status_code=response.status_code)
 
 
-@app.route(drs.drs_http_object_path('{file_uuid}', access_id='{access_id}'), methods=['GET'], cors=True)
+@app.route(drs.drs_http_object_path('{file_uuid}', access_id='{access_id}'), methods=['GET'], cors=True, method_spec={
+    'summary': 'Get a file with an access ID',
+    'description': format_description('''
+        This endpoint returns a URL that can be used to fetch the bytes of a DRS
+        object.
+
+        This method only needs to be called when using an `AccessMethod` that
+        contains an `access_id`
+    ''') + drs_description,
+    'parameters': [
+        params.path('file_uuid', str, description='UUID of the file to be fetched'),
+        params.path('access_id', str, description='Access ID returned from a previous request'),
+    ],
+    'responses': {
+        '301': {
+            'description': format_description('''
+                This response is issued if the object is not yet ready. Respect
+                the `Retry-After` header, then follow the `location` header.
+            '''),
+            'headers': {
+                'Location': {
+                    'description': 'Follow for redirection',
+                    'schema': {'type': 'string', 'format': 'url'},
+                },
+                'Retry-After': responses.header(str, description=format_description('''
+                    Recommended number of seconds to wait before requesting the
+                    URL specified in the Location header.
+                '''))
+            }
+        },
+        '302': {
+            'description': format_description('''
+                The object is ready. Follow the `location` header's URL
+            '''),
+            'headers': {
+                'Location': {
+                    'description': 'Follow for redirection',
+                    'schema': {'type': 'string', 'format': 'url'},
+                }
+            }
+        }
+    },
+    'tags': ['DRS']
+})
 def get_data_object_access(file_uuid, access_id):
     """
```

Comment:
I think we should remove the docstring after making sure it is covered by the spec.